packages/web-client/app/components/card-pay/header/workflow-tracker/item/index.ts

```diff
@@ -14,12 +14,14 @@ import { MILESTONE_TITLES as MERCHANT_CREATION_MILESTONES } from '@cardstack/web
 import { MILESTONE_TITLES as PREPAID_CARD_ISSUANCE_MILESTONES } from '@cardstack/web-client/components/card-pay/issue-prepaid-card-workflow';
 import { MILESTONE_TITLES as RESERVE_POOL_DEPOSIT_MILESTONES } from '@cardstack/web-client/components/card-pay/deposit-workflow';
 import { MILESTONE_TITLES as WITHDRAWAL_MILESTONES } from '@cardstack/web-client/components/card-pay/withdrawal-workflow';
+import { MILESTONE_TITLES as CARD_SPACE_CREATION_MILESTONES } from '@cardstack/web-client/components/card-space/create-space-workflow';
 
 const WORKFLOW_TITLE_TO_MILESTONES: Record<WorkflowName, string[]> = {
   PREPAID_CARD_ISSUANCE: PREPAID_CARD_ISSUANCE_MILESTONES,
   MERCHANT_CREATION: MERCHANT_CREATION_MILESTONES,
   RESERVE_POOL_DEPOSIT: RESERVE_POOL_DEPOSIT_MILESTONES,
   WITHDRAWAL: WITHDRAWAL_MILESTONES,
+  CARD_SPACE_CREATION: CARD_SPACE_CREATION_MILESTONES,
```

Comment:
This makes the workflow tracker show the persisted data properly, but actually restoring is broken… this can be addressed separately though.

![image](https://user-images.githubusercontent.com/43280/136050276-1bb90a3a-b6db-4731-9cfd-7d789fc73db1.png)

It makes me wonder though what the design intentions are here, though… the workflow tracker doesn’t show anywhere in Card Space that I can see. Should people even be able to see/restore Card Space workflows when within Card Pay?

Also semi-related is that when I was exercising this, I found the inability to see what L2 account I had connected confusing… more of a design question as well though, I know you’re implementing it as shown!

AND it doesn’t seem like there‘s a way to get to Card Space once you’ve entered Card Pay and vice versa. I did so by returning to `/` manually but that seems subpar 🤔

Reply:
> This makes the workflow tracker show the persisted data properly, but actually restoring is broken… this can be addressed separately though.
> 
> ![image](https://user-images.githubusercontent.com/43280/136050276-1bb90a3a-b6db-4731-9cfd-7d789fc73db1.png)
> 
> It makes me wonder though what the design intentions are here, though… the workflow tracker doesn’t show anywhere in Card Space that I can see. Should people even be able to see/restore Card Space workflows when within Card Pay?
> 
> Also semi-related is that when I was exercising this, I found the inability to see what L2 account I had connected confusing… more of a design question as well though, I know you’re implementing it as shown!
> 
> AND it doesn’t seem like there‘s a way to get to Card Space once you’ve entered Card Pay and vice versa. I did so by returning to `/` manually but that seems subpar 🤔

Yeah, I was wondering if this would work. Typescript forced me to add this workflow there, but if this tracker is only for Card Pay workflows, then we would need another way to access the Card Space workflow. I'll bring it up to the design channel. For now, I'll try to remove this workflow from the tracker